flink-runtime/src/test/java/org/apache/flink/runtime/security/KerberosUtilsTest.java

```diff
@@ -18,11 +18,11 @@
 
 package org.apache.flink.runtime.security;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import javax.security.auth.login.AppConfigurationEntry;
 
-import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
```

Comment:
Should use the `org.assertj.core.api.Assertions.assertThat`

Reply:
I have replace all asserts with assertj's.